eng/pipelines/eng-common-sync.yml

```diff
@@ -118,75 +110,25 @@ stages:
                           -AuthToken "$(azuresdk-github-pat)"
                           -PRTitle "Sync ${{ parameters.DirectoryToSync }} directory with azure-sdk-tools for PR $(System.PullRequest.PullRequestNumber)"
-                          -PRLabels "Central-EngSys, EngSys"
+                          -PRLabels "Central-EngSys,EngSys,auto-merge"
                           -PRBody "Sync ${{ parameters.DirectoryToSync }} directory with azure-sdk-tools for PR https://github.com/Azure/azure-sdk-tools/pull/$(System.PullRequest.PullRequestNumber)`
                            See [eng/common workflow](https://github.com/Azure/azure-sdk-tools/blob/master/eng/common/README.md#workflow)"
                       ${{ if ne(variables['Build.Reason'], 'PullRequest') }}:
                         arguments: >
                           -RepoOwner "Azure"
                           -RepoName "${{ repo }}"
-                          -BaseBranch $(system.pullRequest.targetBranch)
+                          -BaseBranch ${{ coalesce(parameters.BaseBranchName, $(Build.SourceBranchName)) }}
                           -PROwner "Azure"
                           -PRBranch "sync-${{ parameters.DirectoryToSync }}"
                           -AuthToken "$(azuresdk-github-pat)"
-                          -PRTitle "Sync ${{ parameters.DirectoryToSync }} directory with azure-sdk-tools for PR $(System.PullRequest.PullRequestNumber)"
-                          -PRLabels "Central-EngSys, EngSys"
-                          -PRBody "Sync ${{ parameters.DirectoryToSync }} directory with azure-sdk-tools for PR https://github.com/Azure/azure-sdk-tools/pull/$(System.PullRequest.PullRequestNumber)`
+                          -PRTitle "Sync ${{ parameters.DirectoryToSync }} directory with azure-sdk-tools"
+                          -PRLabels "Central-EngSys,EngSys,auto-merge"
+                          -PRBody "Sync ${{ parameters.DirectoryToSync }} directory with azure-sdk-tools`
                            See [eng/common workflow](https://github.com/Azure/azure-sdk-tools/blob/master/eng/common/README.md#workflow)"
 
+                ${{ if eq(variables['Build.Reason'], 'PullRequest') }}:
                   - pwsh: |
-                      $PRData = "Azure;${{ repo }};$(Submitted.PullRequest.Number)"
-                      Add-Content -Path "$(Build.ArtifactStagingDirectory)/${{ parameters.PRDataFileName }}" -Value $PRData
-                    displayName: Write Sync PR Data to Artifact File
-                    condition: succeeded()
-                
-                - task: PublishPipelineArtifact@1
-                  condition: succeeded()
-                  displayName: Publish ${{ parameters.PRDataFileName }} 
-                  inputs:
-                    artifactName: ${{ parameters.ArtifactName }}
-                    path: $(Build.ArtifactStagingDirectory)/${{ parameters.PRDataFileName }} 
-
-  - stage: VerifyAndMerge
```

Comment:
What blocks someone from merging their Tools PR before all the Sync PR's are merged?

Reply:
I'll add back this stage.